packages/react-native-renderer/src/__tests__/ReactNativeError-test.internal.js

```diff
@@ -32,6 +32,16 @@ describe('ReactNativeError', () => {
         .computeComponentStackForErrorReporting;
   });
 
+  it('should throw error if null component registration getter is used', () => {
+    expect(() => {
+      try {
+        createReactNativeComponentClass('View', null)
+      } catch (e) {
+        throw new Error(e.toString());
+      }
+    }).toThrow('Invariant Violation: View config getter callback must be a function: View (received null)');
```

Comment:
This doesn't read super great to me.

What about:

```
Invariant Violation: Cannot create component View, view config getter must be a function (received null)
```

Reply:
Hmm, that's reasonable but it's the registry that's throwing this error, not the `createReactNativeComponentClass`

Reply:
What about 
```
'Invariant Violation: View config getter callback for component `View` must be a function (received null)'
```